react/components/SKUSelector/utils/index.js

```diff
@@ -41,6 +41,16 @@ export const parseSku = sku => {
   return result
 }
 
+/**
+ * Group the sku items by the variation name specified
+ */
+export const groupItemsByVariation = (name, items) =>
```

Comment:
please, don't be so ramdeee. this is no't js

Reply:
I don't  see a problem. maybe post a example of input and output of that function so we can understand and learn what the function should do

Reply:
we use jsx daily, and it's *literally* not JavaScript, but we don't have any problems using it, why does this bother you?

Reply:
@jgfidelis already answered for me, what's the point of having a code that I have to ready a 'doc' to understand what it does, using ramda this way you are writing basically a black box.

Reply:
Which makes harder to maintain to your peers